packages/react-components/react-shared-contexts/src/index.ts

```diff
@@ -1,4 +1,5 @@
-export { ThemeContext, useTheme } from './ThemeContext';
+export { ThemeContext } from './ThemeContext';
+export type { ThemeContextValue } from './ThemeContext';
```

Comment:
in https://github.com/microsoft/fluentui/pull/23325 we are moving react-shared-contexts to unstable. Does this have any effect on exporting this type from the unstable package?